src/RadixDlt.NetworkGateway.DataAggregator/Configuration/LedgerConfirmationOptions.cs

```diff
@@ -101,6 +101,13 @@ public sealed record LedgerConfirmationOptions
     [ConfigurationKeyName("MaxCommitBatchSize")]
     public long MaxCommitBatchSize { get; set; } = 300;
 
+    /// <summary>
+    /// Gets or sets MinCommitBatchSize.
+    /// The minimum batch to send to the ledger extension service for committing.
```

Comment:
We should probably explain that this is mostly exposed for testing, and we expect 1 is the best value for production use.

Reply:
done